team site and account deletion (#354)
gristlabs/grist-help: help/en/docs/FAQ.md

You can now easily switch between all your accounts, and all your teams, from the user menu.

### How do I delete my account?

> You can delete your account in Profile Settings by selecting Delete Account under Privacy & Data. Please note that this action is permanent. 

Reviewer: Profile Settings, Delete Account & Privacy & Data should be within quotes. Single quotes according to style guide, though on this page they seem to be all double... maybe we can fix that as well.

> Alternatively, you can learn more about deleting team sites [here](https://support.getgrist.com/teams/#billing-account). 

Reviewer 2: ```suggestion
Need to delete a team site? Learn more [here](https://support.getgrist.com/teams/#billing-account). 
```
